projects/uitdatabank/docs/search-api/sorting.md

```diff
@@ -0,0 +1,86 @@
+# Sorting
+
+The UiTdatabank Search API provides several options to sort events, places and organizers.
+
+- `availableTo` 
+- `created`
+- `distance` 
+- `modified`
+- `score`
+
+For every sort parameter it is possible to change the sort `order`:
+
+- Ascending order \(from smallest to biggest value\) by using `asc`
+- Descending order \(from biggest to smallest value\) by using `desc`
+
+## Sort parameters
+### availableTo
+<!-- theme: info -->
+
+> Only applicable on events and places.
+
+Sorts the data by the date until it is available in the API. This is most commonly used to show events & places that will be ending soon first. Sorting on availableTo (ascending) is what we advice for event calendars.
+
+**Example**
+```
+GET https://search.uitdatabank.be/offers/?sort[availableTo]=asc
+```
+Events & places that end soon will appear first, permanent events and places last.
+
+### created
+Sorts the data by the date it is created in the API
+
+**Example**
+```
+GET https://search.uitdatabank.be/organizers/?sort[created]=desc
+```
+Newly created organizers will appear first.
+
+### distance
+<!-- theme: info -->
+
+> Only applicable on events and places, and when the the `coordinates` and `distance` parameter are also present
+
+When searching by `geo distance`, it is possible to sort the results by the distance from the given coördinates.
+
+**Example**
+```
+GET https://search.uitdatabank.be/places/?coordinates=50.8511740,4.3386740&distance=10km&sort[distance]=asc
+```
+Places that are the closest to the given coordinates will appear on top.
+
+
```

Comment:
Double newlines have the potential to make diffs harder to read later. I'll add it to the list of possible checks for the linter to add later. In the meantime it's not super bad if we sometimes have a double (or triple) newline somewhere as Stoplight will not render them anyway, but try to keep an eye out for them if you can to keep the cleanup later limited :)